snuba/schemas.py

```diff
@@ -307,6 +307,26 @@ def validate(self, value) -> Request:
 
         return Request(query, extensions)
 
+    def __generate_template_impl(self, schema) -> Union[Schema, str, None]:
```

Comment:
I think this would need to return `Any`, since the value of `default` on line 318 isn't realistically going to be constrained.

I don't think `Schema` makes too much sense here (or as the return type of `generate_template`, unless this is would be generating a template for a metaschema for some reason), at least by my original intent of what `Schema` was supposed to represent which if made explicit would be something like a `TypedDict` that would enforce the JSON schema metaschema itself. (It should be possible to do, there was just no way I was going to spend the time to actually do it.)

`Schema` should work as the parameter type for `schema`, though.

Reply:
ok, understood